integration-libs/s4om/schematics/add-s4om/index.ts

```diff
@@ -0,0 +1,62 @@
+import {
```

Comment:
Did you tests if those schematics are working with new spartacus app? Are you able to successfully add s4om to the application?

Reply:
I will be testing the schematics soon Michal and will keep you posted